play-java-ebean-example/build.sbt

```diff
@@ -11,10 +11,6 @@ lazy val root = (project in file("."))
       "org.awaitility" % "awaitility" % "3.1.6" % Test,
       "org.assertj" % "assertj-core" % "3.12.2" % Test,
       "org.mockito" % "mockito-core" % "3.0.0" % Test,
-      // To provide an implementation of JAXB-API, which is required by Ebean.
-      "javax.xml.bind" % "jaxb-api" % "2.3.1",
-      "javax.activation" % "activation" % "1.1.1",
-      "org.glassfish.jaxb" % "jaxb-runtime" % "2.3.8",
```

Comment:
Not needed anymore here, see https://github.com/playframework/play-ebean/releases/tag/6.0.0